example/src/App.tsx

```diff
@@ -247,6 +247,25 @@ const AppContainer = () => {
     );
   };
 
+  const handleStopPlayersAndExtractors = async () => {
+    const { stopPlayersAndExtractors } = useAudioPlayer();
+    const hasStoppedAll: boolean[] = await stopPlayersAndExtractors();
```

Comment:
Yes, I will look into it and try!
I intend to use this new method on unmount only, where UI feedback should not matter, but I will put more time into making it more robust!

Reply:
> Can we change start player method to include check if there are waveform only then start playing otherwise first extract waveforms and then start playing.
This is fixed by [this commit](https://github.com/SimformSolutionsPvtLtd/react-native-audio-waveform/pull/119/commits/97cf9f8a2de30f6c497004265bf8fd65b9c59de4)

Reply:
> If we can find a way to do this when the stopAll method gets called, then it will improve the ux of the app.

Trying to get the stopAll to contain the current player scenario might be trickier, and since I'm expecting this to be used on unmount, I would not complicate it by trying to fix it in there.
However, the easiest solution is to stop the current player "manually" and then call stopAll, which would give the experience you requested in the example App. To implement that, I would need a more global handle on the current player as I started in [this PR](https://github.com/SimformSolutionsPvtLtd/react-native-audio-waveform/pull/123). So I can fix the issue once the above PR is merged! I will finish it and get it ready!

Reply:
FYI: I found the icon `tintColor` issue; it was the colour space set to Gray. Once changed to RGB, it now iOS render the `tintColor` correctly

Reply:
Fixed there: https://github.com/SimformSolutionsPvtLtd/react-native-audio-waveform/pull/119/commits/fcc287c4081c0d662ec56261bc99f7a63c6874db